src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentInformationTable.tag

```diff
@@ -6,36 +6,54 @@
 <%@ tag import="teammates.common.util.Const" %>
 <div class="well well-plain">
   <div class="form form-horizontal" id="studentInfomationTable">
-    <div class="form-group">
-      <label class="col-sm-1 control-label">Student Name:</label>
-      <div class="col-sm-11" id="<%=Const.ParamsNames.STUDENT_NAME%>">
-        <p class="form-control-static">${fn:escapeXml(studentInfoTable.name)}</p>
+    <div class="form-group student-info-row">
+      <div class="col-sm-3 text-right">
```

Comment:
from my point of view align right looks pretty weird...once u squish the window/put on a smaller window it will look like this:
![image](https://user-images.githubusercontent.com/28526038/40934941-15e13c34-6869-11e8-9451-0e280536d9a4.png)

Reply:
The problem was that the class wasn't defined for xs devices so it takes up the whole width and titles appear right due to text-right. I have changed the classes to xs.
Looks like this now:
![image](https://user-images.githubusercontent.com/11634699/40975422-c9c41c68-68c2-11e8-86f5-32ad18ea4fa7.png)